doc/annotations.md

```diff
@@ -149,15 +149,16 @@ $evm->addEventSubscriber($sortableListener);
 // mysql set names UTF-8 if required
 $evm->addEventSubscriber(new Doctrine\DBAL\Event\Listeners\MysqlSessionInit());
 // DBAL connection
-$connection = array(
+$conn = array(
```

Comment:
I'd use `$driverConfig`, `$params`, `$settings` or similar in order to avoid the ambiguity between `$conn` and `$connection`.

Reply:
called it $driverParams because $driverConfig would be similar to the second parameter ($config) and the parameter is actually called $params in the method.